### ISSUE TITLE
Kafka Connect - Integrate Sources

### ISSUE THREAD

**Issue by amarziali** [Hurence/logisland#375]
Kafka Connect is a framework included in Apache Kafka that integrates Kafka with other systems. Its purpose is to make it easy to add new systems to your scalable and secure stream data pipelines. (See https://www.confluent.io/product/connectors/).
The basic components a kafka connect connector is made of:

- Connectors – the high-level abstraction that coordinates data streaming by managing tasks
- Tasks – the implementation of how data is copied to or from Kafka
- Workers – the running processes that execute connectors and tasks

It will be interesting to have those source/sink connectors' library off the shelf in logisland.

However, from an architecture point of view, we can see that the scheduling and the distributed herder provided along with kafka connect is somehow "complementary" to what is provided with spark structured streaming.
This means that in logisland we can leverage the StructuredStream processing and use the sources/sink without passing the data into kafka as an intermediate step.

In other words, we are building a kafka connect to logisland adapter, getting rid of kafka connect scheduling part (workers and herders), integrated as controller service, acting as a source/sink of a structured stream.

In this first milestone, only sources will be handled.

**Comment by amarziali** [comment]
PR Merged. Issue closed
